src/dash/vo/Representation.js

```diff
@@ -72,6 +72,7 @@ class Representation {
         this.startNumber = 1;
         this.timescale = 1;
         this.width = NaN;
+        this.dependentRepresentation = null
```

Comment:
all member attributes in this constructor are sorted alphabetically.
Can you move this new entry?